pwa/src/templates/categoriesDetailTemplate/CategoriesDetailTemplate.tsx

```diff
@@ -30,12 +32,8 @@ export const CategoriesDetailTemplate: React.FC<CategoriesDetailTemplateProps> =
     TEMPORARY_DOMAINS.find((domain) => {
       return domain.title === portfolio.domain;
     });
-  const component =
-    portfolio &&
-    TEMPORARY_COMPONENTS.find((component) => {
-      return component.id === "aa78c93b-d57d-4d9a-b255-e8cfe26bde48";
-    });
-  const badgeNumber = 1;
+  const components = TEMPORARY_COMPONENTS;
```

Comment:
Shouldn't this just be the components that are filtered for the current category? See previous comment. 

If the test data doesn't allow it, just keep using `TEMPORARY_COMPONENTS` throughout the file. No need to do this one to one assignment.